src/components/qaDatatablesContainer/QALinearitySummaryExpandableRows/QALinearitySummaryExpandableRows.js

```diff
@@ -1,405 +1,32 @@
-            (<QADataTableRender
-              columnNames={columns}
-              columnWidth={15}
-              data={[]}
-              actionColumnName={
-                user ? (
-                  <>
-                    <span className="padding-right-2">Test Data</span>
-                    <Button
-                      epa-testid="btnOpen"
-                      className="text-white"
-                      onClick={() => openModal(false, false, true)}
-                    >
-                      Add
-                    </Button>
-                  </>
-                ) : (
-                  "Test Data"
-                )
-              }
-              actionsBtn={"View"}
-              user={user}
-            />) : "There're no records available."
-          }
-        />
-      ) : (
-        <Preloader />
-      )}
-
-      {show ? (
-        <Modal
-          show={show}
-          close={closeModalHandler}
-          save={createNewData ? createData : saveData}
-          showCancel={!user || nonEditable}
-          showSave={user && !nonEditable}
-          nonEditable={nonEditable}
-          title={
-            createNewData
-              ? `Add  ${dataTableName}`
-              : user
-              ? ` Edit ${dataTableName}`
-              : ` ${dataTableName}`
-          }
-          exitBTN={createNewData ? `Create ${dataTableName}` : `Save and Close`}
-          children={
-            dropdownsLoaded ? (
-              <div>
-                <ModalDetails
-                  modalData={selectedRow}
-                  data={selectedModalData}
-                  // prefilteredMdmData={prefilteredMdmData}
-                  cols={3}
-                  title={`${dataTableName}`}
-                  viewOnly={!user || nonEditable}
-                  create={createNewData}
-                  // setMainDropdownChange={setMainDropdownChange}
-                  //mainDropdownChange={mainDropdownChange}
-                  // onEditUpdateHandler={onEditUpdateHandler}
-                />
-              </div>
-            ) : (
-              <Preloader />
-            )
-          }
-        />
-      ) : null}
-    </div>
-  );
-};
-const mapStateToProps = (state, ownProps) => {
-  const dataTableName = "Linearity Test";
-  return {
-    mdmData: state.dropdowns[convertSectionToStoreName(dataTableName)],
-  };
-};
-
-const mapDispatchToProps = (dispatch) => {
-  return {
-    loadDropdownsData: async (section, dropdownArray) =>
-      dispatch(
-        loadDropdowns(convertSectionToStoreName(section), dropdownArray)
-      ),
-  };
-};
-
-export default connect(
-  mapStateToProps,
-  mapDispatchToProps
-)(QALinearitySummaryExpandableRows);
-export { mapDispatchToProps };
-export { mapStateToProps };
+    <>
+      <LinearitySummaryDataRows
+        user={user}
+        nonEditable={nonEditable}
+        locationSelectValue={locationSelectValue}
+        data={data}
+      />
+      <ProtocolGasDataRows
+        user={user}
```

Comment:
i don't think we need a separate component to render both LinearitySummaryDataRows and ProtocolGasDataRows as you can just import and throw it in QALinearitySummaryExpandableRows